test/configCases/web/node-source/index.mjs

```diff
@@ -0,0 +1 @@
+global
```

Comment:
You also need to add a test like this:

``` js
it("should compile fine", () => {
  global
});
```

Reply:
Done, and I added another line to better test what I was going for.